staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -130,6 +132,9 @@ type GenericAPIServer struct {
 	swaggerConfig *swagger.Config
 	openAPIConfig *openapicommon.Config
 
+	// Enables updating OpenAPI spec using update method.
+	OpenAPIService *openapi.OpenAPIService
```

Comment:
More on this from another comment by @sttts:

> Static in which sense? Resources can come and go. With CRD validation, we will also get JSON Schema snippets that can change any moment and should be part of the OpenAPI spec served by the aggregator.

I think not only we need this, we need an upstream channel to tell aggregator it changed (maybe something like a watch on OpenAPI spec).